source/_posts/2017/2017-01-31-how-monolithic-repository-in-open-source-saved-my-lazyness.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: "How Monolithic Repository in Open Source saved my Lazyness"
+perex: '''
+    I've started creating open-source about 6 years ago. Now I'm maintaining over 20 repositories. I used classic standalone
+    repositories, but with each new package I wanted to add I realized, how much work it needs to keep everything up-to-date and consistent. So I didn't and got stuck.
+    Fortunately, I noticed talk from Fabien about some *monorepo* and Symfony. I said to myself: "I don't know a think about it. Let's try it out. I can return if it sucks."
+    <br><br>
+    I never dit. Today I will show you <strong>why I see monorepo approach in open-source so awesome</strong>.
+'''
+lang: en
+---
+
+If you have 50 minutes, [check the talk about Symfony monorepo](https://www.youtube.com/watch?v=4w3-f6Xhvu8) by Fabien. If you don't (my case), here are **[40 slides](https://speakerdeck.com/fabpot/a-monorepo-vs-manyrepos) from the talk**.
+
+The talk references a tool, that helps you with splits - [splitsh](https://github.com/splitsh/lite). Do you understand it? Me neither.
+Splitsh a complex tool to maintain Symfony and Blackfire ecosystem. **All we need is one git command**.
```

Comment:
Možná stojí za zdůraznění to, že subtree je narozdíl od Splitsh hooodně pomalý - proto se v Symfony nepoužívá.